arcgis-ios-sdk-samples/Content Display Logic/Controllers/SampleInfoViewController.swift

```diff
@@ -32,24 +32,26 @@ class SampleInfoViewController: UIViewController {
         view.addSubview(webView)
         self.webView = webView
         
-        if self.folderName != nil {
-            self.fetchFileContent(for: self.folderName)
+        if let readmeURL = readmeURL,
+            let html = markdownTextFromFile(at: readmeURL) {
+            displayHTML(html)
         }
     }
     
-    func fetchFileContent(for folderName:String) {
+    func markdownTextFromFile(at url:URL)->String? {
```

Comment:
Please, PLEASE put some more spaces in there! The colon (`:`) and the arrow (`->`) get lost without spaces. The rule we try to follow is there should always be a space (` `) after every colon (anywhere, not just in a function signature) and there should be a space before and after the arrow.